packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -95,7 +95,9 @@ function PostLink( {
 					/>
 					<p>
 						{ __( 'The last part of the URL.' ) }{ ' ' }
-						<ExternalLink href="https://wordpress.org/support/article/writing-posts/#post-field-descriptions">
+						<ExternalLink
+							href={ __( 'https://wordpress.org/support/article/writing-posts/#post-field-descriptions' ) }
```

Comment:
```suggestion
							href={ __(
								'https://wordpress.org/support/article/writing-posts/#post-field-descriptions'
							) }
```

fix for code formatting